job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext.groovy

```diff
@@ -154,6 +155,40 @@ class TriggerContext extends AbstractExtensibleContext {
                 allowMembersOfWhitelistedOrgsAsAdmin pullRequestBuilderContext.allowMembersOfWhitelistedOrgsAsAdmin
             }
         }
+        if (!jobManagement.getPluginVersion('ghprb')?.isOlderThan(new VersionNumber('1.22-0'))) {
+            Node extensionsNode = new NodeBuilder().'extensions' {}
```

Comment:
The NodeBuilder allows to create nested elements as I mentioned before. Use that instead of creating separate nodes for sub-elements.